autosar/writer/portinterface_writer.py

```diff
@@ -1,3 +1,4 @@
+import autosar.element

```

Comment:
let's directy import `AutosarDataPrototype` to avoid much verbosity to access the `Roles`

```suggestion
from autosar.element import AutosarDataPrototype
```

NOTE, the same applies to other files as well